README.md

```diff
@@ -46,6 +46,27 @@ platforms will be officially supported after release `1.0`.
 The latest version of neofs-node works with neofs-contract 
 [v0.13.0](https://github.com/nspcc-dev/neofs-contract/releases/tag/v0.13.0).
 
+# Building
+
+To make all binaries you need Go 1.17+ and `make`:
+```
+make build
```

Comment:
we do not have that target, mb `make all`?

Reply:
Fixed.